h2o-core/src/main/java/water/H2O.java

```diff
@@ -1550,8 +1554,26 @@ static public void finalizeRegistration() {
   // A dense array indexing all Cloud members. Fast reversal from "member#" to
   // Node.  No holes.  Cloud size is _members.length.
   public final H2ONode[] _memary;
+
+  // mapping from a node ip to node index
+  private final HashMap<String, Integer> node_ip_to_index;
```

Comment:
Minor comment: to be consistent with rest of code, the name should start with `_` :-|